sql/moz-fx-data-shared-prod/search_derived/mobile_search_clients_daily_v1/query.sql

```diff
@@ -260,7 +260,7 @@ metrics_org_mozilla_ios_firefox AS (
     DATE(submission_timestamp) AS submission_date,
     client_info.client_id,
     normalized_country_code AS country,
-    'Fennec' AS app_name,
```

Comment:
Is this dataset the only one "Fennec" is being used in?

Reply:
@alekhyamoz we should probably check with @scholtzan what happens to the normalized app name if you haven't already, given that this might have downstream consequences.

Reply:
Checked with @scholtzan  and it was confirmed that `normalized_app_name` can remain 'Fennec'

Reply:
Thank you for checking!